content/developers/internals/data-model/usage-example.en.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Usage Example"
+linkTitle: "Example"
+weight: 20
+description: "OSRD data format usage example"
+---
+
+## Introduction
+
+This page gives an example of how the data formats are used to describe an infrastructure in *OSRD*.
+
+For this purpose, let's take as an example the following toy infrastructure:
+
+![Toy infrastructure schema](../svg_schemas/small_infra_schema.drawio.en.svg)
+
+{{% alert title="Tip" color="info"%}}
+In case you want to zoom in on schemas, you can click on the little editing button that appears when hovering the mouse on it.
+{{% /alert %}}
+
+In this schema we represent the overall look of the infrastructure with just the lines and the stations.
+
+This is obviously not a realistic infrastructure, but it will allow us to present the data models in a simple way.
+
+We will build the details of the infrastructure step by step, and present the objects used at each step.
+
+### The infrastructure generator
+
+In the [*OSRD* repository](https://github.com/DGEXSolutions/osrd) is a [python module](https://github.com/DGEXSolutions/osrd/tree/dev/core/examples/generated) that allows for generating infrastructures in a format understood by *OSRD*.
+
+The infrastructure we talk about in this page can be generated through the [small_infra.py](https://github.com/DGEXSolutions/osrd/blob/dev/core/examples/generated/scripts/small_infra.py) file. For more information about the generation scripts, you can check out the related [README](https://github.com/DGEXSolutions/osrd/blob/dev/core/examples/generated/README.md).
+
+## The rails
```

Comment:
I fear that this might be confusing for newcomers, or maybe if we call the section graph, we could add a paragraph about why an infrastructure can be seen as a graph ?